Automattic/wp-calypso
# Refactor/minimum currency amounts

Related to https://github.com/Automattic/gold/issues/91

## Proposed Changes

* This change will pull the minimum currency amounts from the Memberships library on WPCOM rather than store it in Calypso, negating the need to update in multiple locations should those amounts change in the future.

## Testing Instructions


1. Switch to this branch.
2. Additionally, patch D118035-code
3. On a site with an appropriate plan, go to Tools > Earn, then click Manage Payment Button.
4. Click to add a new payment plan.
5. Ensure that the currency list loads, and if you attempt to use a lower amount than listed for the currency in `class.membership-product.php`, you should get an error. 
6. Attempt to add a plan at the minimum for the currency; it should create the plan as expected.

![Screen Shot 2023-08-27 at 2 16 18 PM](https://github.com/Automattic/wp-calypso/assets/8002138/ca73a0cd-132e-4662-8fba-1fa0e6130456)

## Pre-merge Checklist
